src/cephadm/cephadm

```diff
@@ -12,8 +12,9 @@ CONTAINER_PREFERENCE = ['podman', 'docker']  # prefer podman to docker
 CUSTOM_PS1=r'[ceph: \u@\h \W]\$ '
 DEFAULT_TIMEOUT=None # in seconds
 DEFAULT_RETRY=10
-SHELL_DEFAULT_CONF='/etc/ceph/ceph.conf'
-SHELL_DEFAULT_KEYRING='/etc/ceph/ceph.client.admin.keyring'
+ETC_CEPH_DIR="/etc/ceph"
```

Comment:
or should we name this SHELL_DEFAULT_CONF_DIR? What's the purpose of it?

Reply:
Feels like it should be other way around? Rename `SHELL_DEFAULT_CONF` -> `ETC_CEPH_CONF` ?